lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2842,4 +2842,161 @@ describe('AwsCompileFunctions #2', () => {
         );
     });
   });
+
+  describe('when using fileSystemConfigs', () => {
+    it('should apply changes when single fileSystemConfig provided', () => {
+      const arn =
+        'arn:aws:elasticfilesystem:region:111111111111:access-point/fsap-a1a1a1a1a1a1a1a1a';
+      const localMountPath = '/mnt/path';
+      runServerless({
```

Comment:
We should return the result of `runServerless` otherwise orphaned async flow is created, and it might be responsible for crash in tests

Reply:
thanks - I totally missed that and it seems like it might be the cause of errors :crossed_fingers:

Reply:
That was exactly the case, thanks again :raised_hands: